src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -13,6 +13,10 @@ case class ThurloeKeyValue(
   userId: Option[String] = None,
   keyValuePair: Option[FireCloudKeyValue] = None)
 
+case class ThurloeKeyValues(
```

Comment:
Similar to my other comment, `UserMap` might be a better name here, since it's basically just a map (a sequence of key/value pairs) tied to a userId. An optional userId?